packages/lame/lame.0.3.7/opam

```diff
@@ -0,0 +1,39 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "MP3 encoding library"
+description:
+  "Bindings for the lame library which provides functions for encoding mp3 files"
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+license: "GPL-2.0"
```

Comment:
```suggestion
license: "GPL-2.0-only"
```
Or
```suggestion
license: "GPL-2.0-or-later"
```
Or somethelse?

Reply:
Thanks! Changes applied here and upstream.